internal/liqonet/tunnel-operator/tunnel-operator.go

```diff
@@ -165,23 +172,23 @@ func (tc *TunnelController) Reconcile(ctx context.Context, req ctrl.Request) (ct
 		// Set cluster tunnel as ready
 		tc.readyClustersMutex.Lock()
 		defer tc.readyClustersMutex.Unlock()
-		tc.readyClusters[tep.Spec.ClusterID] = struct{}{}
+		tc.readyClusters[clusterIdentity.ClusterID] = struct{}{}
```

Comment:
```suggestion
		tc.readyClusters[tep.Spec.ClusterIdentity.ClusterID] = struct{}{}
```

To me, it looks definitely less confusing and error prone, given the fact it is initialized below. Same for all occurrences.